doc-content/enterprise-only/getting-started/business-process-proc.adoc

```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
 
 .Procedure
-. Click on the start event and create an outgoing connection from the start event to an exclusive gateway.
+
+. Create an outgoing connection from the start event to an exclusive gateway:
+.. Click the start event and click the *Create Parallel* icon.
++
+image::getting-started/create-parallel.png[]
+.. Hover over the image:getting-started/config-icon.png[] icon of the parallel and click the *Convert into Exclusive* icon.
 +
 image::getting-started/exclusive-gateway.png[]
 
-. Create an outgoing connection from the exclusive gateway to a task.
+. Create an outgoing connection from the exclusive gateway to a business rule task:
+.. Click the exclusive gateway and click the *Create Task* icon.
 +
 image::getting-started/create-task.png[]
-
-. Convert the new task to a business rule task.
+.. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/bus-rule-task.png[]
 
-. Click on the business rules task and enter `Validation` in the *Name* field of the *Diagram properties* panel.
-. Expand *Implementation/Execution*, select *validation* in the *Rule Flow Group* field.
-. Define the following Java expression in the *On Exit Action* field :
+. Configure the business rule task:
+
+.. Click the business rules task.
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Validation` in the *Name* field.
+.. Expand *Implementation/Execution* and from the *Rule Flow Group* list select *validation*.
+.. In the *On Exit Action* field, enter the following Java expression:
 +
 [source,java]
 ----
 System.out.println(application.getProperty());
 ----
-+
-. Scroll down and expand the *Data Assignments* section and click image:getting-started/btn_assign.png[] next to *Assignments*.
-. In the *Validation Data I/O* window, click *Add* to create the following assignments:
+.. Expand *Data Assignments* and click image:getting-started/btn_assign.png[] next to *Assignments*.
+.. In the *Validation Data I/O* window, click *Add* and create the following assignments:
 +
 image::getting-started/val-data-io.png[]
 
-. Click *Save*.
-. In the diagram editor, click *Save*, and *Save*, to confirm your changes.
+.. Click *Save*.
+. In the diagram editor, click *Save* and click *Save* again to confirm your changes.
 
 == Defining the validation data
-This section describes how to define the data that determines whether the application data is correct or contains an error or any missing information.
+This section shows you how to define the validation data that determines whether the application data is correct, contains an error, or has missing information.
```

Comment:
Does the data distinguish between errors and missing information? Does it also tell you how to fix it?

Reply:
This depends on how the process developer defines the validation data.

Reply:
So basically the process developer can specify if the validation date  can do any of these functions?

Reply:
yes, from my understanding, the developer can create criteria very roughly equivalent to something like these:
loan amount =! null - if false then return loan amount error 
loan amount > 0 - if false then return message "loan amount has to be greater than 0"
date of birth older than (today's date - 18 years) - if false then return message "applicant is under 18"